### ISSUE TITLE
Make check fail after complete installation

### ISSUE THREAD

**Issue by SamirBitzees** [dogecoin/dogecoin#3280]
```
ubuntu@ip-172-31-14-216:~/dogecoin$ make check
Making check in src
make[1]: Entering directory '/home/ubuntu/dogecoin/src'
make[2]: Entering directory '/home/ubuntu/dogecoin/src'
make[3]: Entering directory '/home/ubuntu/dogecoin'
make[3]: Leaving directory '/home/ubuntu/dogecoin'
make[3]: Entering directory '/home/ubuntu/dogecoin/src/secp256k1'
make[3]: Leaving directory '/home/ubuntu/dogecoin/src/secp256k1'
make  check-TESTS check-local
make[3]: Entering directory '/home/ubuntu/dogecoin/src'
make[4]: Entering directory '/home/ubuntu/dogecoin/src'
make[5]: Entering directory '/home/ubuntu/dogecoin'
make[5]: Leaving directory '/home/ubuntu/dogecoin'
make[5]: Entering directory '/home/ubuntu/dogecoin/src/secp256k1'
make[5]: Leaving directory '/home/ubuntu/dogecoin/src/secp256k1'
FAIL: test/test_dogecoin
============================================================================
Testsuite summary for Dogecoin Core 1.14.6
============================================================================
# TOTAL: 1
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See src/test-suite.log
Please report to https://github.com/dogecoin/dogecoin/issues
============================================================================
make[4]: *** [Makefile:9925: test-suite.log] Error 1
make[4]: Leaving directory '/home/ubuntu/dogecoin/src'
make[3]: *** [Makefile:10033: check-TESTS] Error 2
make[3]: Leaving directory '/home/ubuntu/dogecoin/src'
make[2]: *** [Makefile:10138: check-am] Error 2
make[2]: Leaving directory '/home/ubuntu/dogecoin/src'
make[1]: *** [Makefile:9817: check-recursive] Error 1
make[1]: Leaving directory '/home/ubuntu/dogecoin/src'
make: *** [Makefile:693: check-recursive] Error 1
```

**Comment by SamirBitzees** [comment]
> openssl version

ubuntu@ip-172-31-14-216:~/dogecoin$ openssl version
OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)

**Comment by SamirBitzees** [comment]
> What you can do is cherry pick [6f21685](https://github.com/dogecoin/dogecoin/commit/6f216855bd0730e278de19144ca85d8157a9c02d) on top of the v1.14.6 tag:
> 
> ```shell
> make distclean # only if you kept the results from the previous compile
> git checkout -b my-patched-1.14.6 v1.14.6 # make a new branch to not get a lot of complaints later
> git cherry-pick 6f216855bd0730e278de19144ca85d8157a9c02d # pick the patch
> # redo the build.. (start with ./autogen.sh)



Thank You now my make check execute without Fail.....

**Comment by SamirBitzees** [comment]
I have one more I an week ago raise but not resolve and recently also I posted that #3283 , I unable connect node from backend fail to connect  can you please help me, I am not getting response from any other community sources. I am trying to fixed these issue from one week....
